net/vpnbypass/files/vpnbypass.init

```diff
@@ -1,9 +1,9 @@
 #!/bin/sh /etc/rc.common
 
-START=92
+START=99
```

Comment:
Then you might add there a "sleep 3" to the beginning of the init script, so that network and related services would have a bit more time to start before this one runs. That has been used as cure sometimes.

Network starts asynchronously, so having 99 or 94 makes no real difference, unless you are starting something relevant with 95-98. You might check your /etc/rc.d contents.